sdk/face/ai-vision-face-rest/test/public/faceSessionClient.spec.ts

```diff
@@ -0,0 +1,562 @@
+  ];
+
+  let recorder: Recorder;
+  let client: FaceClient;
+
+  beforeEach(async (context) => {
+    recorder = await createRecorder(context);
+    client = await createClient(recorder);
+  });
+
+  afterEach(async () => {
+    if (recorder?.recordingId) {
+      await recorder.stop();
+    }
+  });
+
+  it("TestDeleteAllVerifySessions", async () => {
+    const getLivenessSessionsResponse = await client
+      .path("/detectLivenessWithVerify/singleModal/sessions")
+      .get();
+    if (isUnexpected(getLivenessSessionsResponse)) {
+      throw new Error(getLivenessSessionsResponse.body.error.message);
+    }
+    assert.equal(getLivenessSessionsResponse.status, "200");
+    console.log(`Number of sessions: ${getLivenessSessionsResponse.body.length}`);
+    const deleteLivenessSessionResponses = await Promise.all(
+      getLivenessSessionsResponse.body.map((livenessSessionItemOutput) =>
+        client
+          .path(
+            "/detectLivenessWithVerify/singleModal/sessions/{sessionId}",
+            livenessSessionItemOutput.id,
+          )
+          .delete(),
+      ),
+    );
+    assert.isTrue(deleteLivenessSessionResponses.every((response) => response.status === "200"));
+  });
+
+  it("TestCreateVerifySession", async () => {
+    const deviceCorrelationId = recorder.variable("deviceCorrelationId", randomUUID());
+    const createLivenessSessionResponse = await client
+      .path("/detectLivenessWithVerify/singleModal/sessions")
+      .post({
+        body: {
+          livenessOperationMode: "Passive",
+          deviceCorrelationId,
+        },
+      });
+    if (isUnexpected(createLivenessSessionResponse)) {
+      throw new Error(createLivenessSessionResponse.body.error.message);
+    }
+    assert.equal(createLivenessSessionResponse.status, "200");
+    assert.isNotEmpty(createLivenessSessionResponse.body.sessionId);
+    assert.isNotEmpty(createLivenessSessionResponse.body.authToken);
+
+    const { sessionId } = createLivenessSessionResponse.body;
+
+    const getLivenessSessionResultResponse = await client
+      .path("/detectLivenessWithVerify/singleModal/sessions/{sessionId}", sessionId)
+      .get();
+    if (isUnexpected(getLivenessSessionResultResponse)) {
+      throw new Error(getLivenessSessionResultResponse.body.error.message);
+    }
+    assert.equal(getLivenessSessionResultResponse.status, "200");
+    assert.equal(getLivenessSessionResultResponse.body.deviceCorrelationId, deviceCorrelationId);
+
+    const deleteLivenessSessionResponse = await client
+      .path("/detectLivenessWithVerify/singleModal/sessions/{sessionId}", sessionId)
+      .delete();
+    if (isUnexpected(deleteLivenessSessionResponse)) {
+      throw new Error(deleteLivenessSessionResponse.body.error.message);
+    }
+    assert.equal(deleteLivenessSessionResponse.status, "200");
+  });
+
+  // TODO: readFileSync is not available in browser environment.
+  // it.runIf(isLiveMode())("TestCreateVerifySessionWithImage", async () => {
```

Comment:
@chungshengfu Just confirm
- Can this case pass in record mode?
- Can this case pass in playback mode?

Reply:
Just tested this. Both record and playback modes passed in Node env.